Spanner/src/Database.php

```diff
@@ -413,6 +418,15 @@ public function drop(array $options = [])
         $this->connection->dropDatabase($options + [
             'name' => $this->name
         ]);
+
+        if ($this->sessionPool) {
+            $this->sessionPool->clear();
+        }
+
+        if ($this->session) {
```

Comment:
should we done prior to the sessionPool->clear() ?
sessionPool clears the cache & seems to be deleting all sessions too